### PR TITLE
PXP-2338 Fix/export encode

### PR DESCRIPTION
**Pull request by paulineribeyre**
Bug fixes in the export functions

### Bug Fixes
- fix encoding bug in sheepdog multiple node export
- fix asterisk bug in sheepdog single node export

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
Jenkins Build 1 : time taken 41 min

**Comment by PlanXCyborg** [comment]
Jenkins Build 2 : time taken 43 min

**Comment by PlanXCyborg** [comment]
Jenkins Build 3 : time taken 26 min

**Comment by PlanXCyborg** [comment]
Jenkins Build 5 : time taken 1 hr 0 min

**Comment by PlanXCyborg** [comment]
Jenkins Build 6 : time taken 25 min

File: tests/integration/datadict/submission/test_endpoints.py [hunk 1]
```diff
@@ -640,3 +642,50 @@ def test_export_all_node_types_json(client, pg_driver, cgci_blgsp, submitter):
...
+        headers=submitter)
+    assert len(r.json) == 1
+
+    # TSV multiple node export
+    path = '/v0/submission/CGCI/BLGSP/export/?node_label=experiment'
+    r = client.get(
+        path,
+        headers=submitter)
+    assert r.status_code == 200, r.data
+    assert r.headers['Content-Disposition'].endswith('tsv')
```
Comment (rudyardrichter) [inline tests/integration/datadict/submission/test_endpoints.py]: could you add this to the test here?
```python
import csv
from StringIO import StringIO


tsv_output = csv.DictReader(StringIO(rs.data), delimiter="\t")
row = next(tsv_output)
assert row["submitter_id"] == "BLGSP-submitter-ü"
```

**Comment by PlanXCyborg** [comment]
Jenkins Build 7 : time taken 26 min

**Pull request merged**